ingestion/setup.py

```diff
@@ -20,16 +20,19 @@
 # Add here versions required for multiple plugins
 VERSIONS = {
     "airflow": "apache-airflow==2.7.3",
+    "adlfs": "adlfs==2022.11.2",  # Python 3.7 does only support up to 2022.2.0
```

Comment:
we don't support python 3.7 anymore so I guess we can just remove the comment

Reply:
also might be better to keep things a bit more generic with `~=` in case people install this on their instances